Microsoft.DotNet.Interactive.Tests/LanguageKernelFormattingTests.cs

```diff
@@ -89,6 +89,34 @@ public LanguageKernelFormattingTests(ITestOutputHelper output) : base(output)
                                    v.Value.ToString().Contains(expectedContent));
         }
 
+        [Theory]
+        [InlineData(Language.CSharp, "new LaTeX(@\"F(k) = \\int_{-\\infty}^{\\infty} f(x) e^{2\\pi i k} dx\")", @"F(k) = \int_{-\infty}^{\infty} f(x) e^{2\pi i k} dx")]
+        [InlineData(Language.CSharp, "display(new LaTeX(@\"F(k) = \\int_{-\\infty}^{\\infty} f(x) e^{2\\pi i k} dx\"));", @"F(k) = \int_{-\infty}^{\infty} f(x) e^{2\pi i k} dx")]
+        [InlineData(Language.FSharp, "new LaTeX(@\"F(k) = \\int_{-\\infty}^{\\infty} f(x) e^{2\\pi i k} dx\")", @"F(k) = \int_{-\infty}^{\infty} f(x) e^{2\pi i k} dx")]
+        [InlineData(Language.FSharp, "display(new LaTeX(@\"F(k) = \\int_{-\\infty}^{\\infty} f(x) e^{2\\pi i k} dx\"))", @"F(k) = \int_{-\infty}^{\infty} f(x) e^{2\pi i k} dx")]
+        public async Task LaTeX_is_rendered_as_text_latex(
+            Language language,
+            string submission,
+            string expectedContent)
+        {
+            var kernel = CreateKernel(language);
+
+            var result = await kernel.SendAsync(new SubmitCode(submission));
+
+            var valueProduced = await result
```

Comment:
This await and timeout pattern should be avoided. It's a bit verbose and it can hang the test run if someone forgets the timeout trivia.

Do this instead:

```csharp
KernelEvents
  .Should()
  .ContainSingle<DisplayEventBase>()
  .Which
  .FormattedValues
  .Should()
                .ContainSingle(v =>
                    v.MimeType == "text/latex" &&
                    v.Value.ToString().Contains(expectedContent));
```